frontend/src/main/pages/CourseSearchPage.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import CourseSearchForm from "main/components/CourseSearch/CourseSearchForm";
+
+import { useBackend } from "main/utils/useBackend";
```

Comment:
It says this is an unused import, unless we're planning to use it later on.

Reply:
Seconded

Reply:
Removed it, at the time I figured we might need it when the backend stuff was ready